OasisDEX/oasis-borrow
# Fix background scroll

# [Fix background scroll](https://app.shortcut.com/oazo-apps/story/4956/fix-background-getting-cut-off)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
background is passsed into the Layout objets as props 
## How to test 🧪
  <Please explain how to test your changes>
- scroll to the bottom and check the footer

![image](https://user-images.githubusercontent.com/6533433/184132988-0dff5da5-c5f3-4770-a7e8-04a9450f78df.png)


- check that the bg is scrolling with the page as in Figma:
- 
![image](https://user-images.githubusercontent.com/6533433/184132909-e873dfed-4752-47fb-a22e-8b31bbd7ea18.png)
